src/App.css

```diff
@@ -1,3 +1,49 @@
+.container {
```

Comment:
is this not available in the dotgov css?

Reply:
pr https://github.com/baltimorecounty/react-services-list/pull/11 addresses this particular issue.